### PR TITLE
Fan-Made Scenarios: Addition of memory functionality, coordinate cleaning of upper table

### PR DESCRIPTION
**Pull request by Chr1Z93**
image

### CODE REVIEW

File: objects/Fan-MadeScenariosCampaignsMiscellany.66e97c.ttslua [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+    for guid, entry in pairs(memoryList) do
+        memoryList[guid].pos.x = entry.pos.x - deltaPos.x
+        memoryList[guid].pos.y = entry.pos.y - deltaPos.y
+        memoryList[guid].pos.z = entry.pos.z - deltaPos.z
+        -- memoryList[guid].rot.x = movedRotation.x
+        -- memoryList[guid].rot.y = movedRotation.y
+        -- memoryList[guid].rot.z = movedRotation.z
+    end
+
+    --theList[obj.getGUID()] = {
```
Comment (Buhallin) [inline objects/Fan-MadeScenariosCampaignsMiscellany.66e97c.ttslua:44]: Any reason not to clean this up?

**Review by Buhallin — APPROVED** [review]
I'm going to assume the copy-paste on the memory bag is good.

Two general comments:
1. Please separate PRs functionally, we again have two completely different issues mixed
2. More detailed description would be helpful when reviewing.  What's it doing on a functional level.  Are the coordinate changes for alignment?  Prettifying?  Was there an actual issue?

**Comment by Chr1Z93** [comment]
> I'm going to assume the copy-paste on the memory bag is good.
> 
> Two general comments:
> 1. Please separate PRs functionally, we again have two completely different issues mixed
> 2. More detailed description would be helpful when reviewing.  What's it doing on a functional level.  Are the coordinate changes for alignment?  Prettifying?  Was there an actual issue?

Yeah, this could have been split into two PRs.
For future reference: this was just alignment into the existing positions.

File: objects/Fan-MadeScenariosCampaignsMiscellany.66e97c.ttslua (hunk 1 above)
Comment (Chr1Z93) [reply objects/Fan-MadeScenariosCampaignsMiscellany.66e97c.ttslua:44]: Code is copy&paste from the official scenario box to the left of it.

If I were to look into it, I'd probably remove a lot of code from it 😅

**Pull request merged**